cypress/e2e/pages/errorPage404/index.cy.js

```diff
@@ -1,13 +1,150 @@
-import runTestsForPage from '../../../support/helpers/runTestsForPage';
-import { testsThatFollowSmokeTestConfig } from './tests';
-import { testsThatFollowSmokeTestConfigForAMPOnly } from './testsForAMPOnly';
-import { testsThatFollowSmokeTestConfigForCanonicalOnly } from './testsForCanonicalOnly';
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
+import testsThatFollowSmokeTestConfig from './tests';
+import testsThatFollowSmokeTestConfigForAMPOnly from './testsForAMPOnly';
+import testsThatFollowSmokeTestConfigForCanonicalOnly from './testsForCanonicalOnly';
```

Comment:
Let's rename these imports:

```suggestion
import testsForAllPages from './tests';
import ampTests from './testsForAMPOnly';
import canonicalTests from './testsForCanonicalOnly';
```